context/app/static/js/components/Providers.jsx

```diff
@@ -49,4 +52,5 @@ Providers.propTypes = {
 };
 
 export { AppContext };
+export const useFlaskDataContext = () => useContext(FlaskDataContext);
```

Comment:
With the refactoring to AppContext mentioned above, I'd prefer to have both of these moved to a separate file.

Reply:
Okay, I will create a separate file for Context